### PR TITLE
MCKIN-12672 Mobile: Fix seekbar, seeker discrepancy

### PR DESCRIPTION
**Pull request by moeez96**
Jira ticket: https://edx-wiki.atlassian.net/browse/MCKIN-12672

Fixed by pausing,  playing video on fullscreen change

### CODE REVIEW

File: ooyala_player/public/js/ooyala_player.js [hunk 1]
```diff
@@ -183,6 +183,17 @@ function OoyalaPlayerBlock(runtime, element) {
                     time: this.player.getPlayheadTime(),
                     player_state: this.player.getState()
                 });
+                console.log('fullscreenchanged')
```
Comment (naeem91) [inline ooyala_player/public/js/ooyala_player.js]: Please remove the debug messages

File: ooyala_player/public/js/ooyala_player.js [hunk 2]
```diff
@@ -183,6 +183,17 @@ function OoyalaPlayerBlock(runtime, element) {
...
+                var playing = this.player.isPlaying();
+                console.log('playing: ' + playing)
+                if (playing == true) {
```
Comment (naeem91) [inline ooyala_player/public/js/ooyala_player.js]: how is this functionality related to fullscreen event?

File: ooyala_player/public/js/ooyala_player.js [hunk 3]
```diff
@@ -183,6 +183,17 @@ function OoyalaPlayerBlock(runtime, element) {
...
+                  $('#' + this.data.domId).find('.oo-icon-pause-slick').click()
+                  setTimeout(() => {$('#' + this.data.domId).find('.oo-icon-play-slick').click()}, 0.3*1000)
```
Comment (naeem91) [inline ooyala_player/public/js/ooyala_player.js]: did we confirm that arrow function is transpiling to plain javascript?

File: ooyala_player/public/js/ooyala_player.js (hunk 2 above)
Comment (moeez96) [reply ooyala_player/public/js/ooyala_player.js]: Dsicrepency only occurs at fullscreen change
Check at https://stage.mckinsey.edx.org/courses/UI/Refresh/2019/lessons/i4x://UI/Refresh/chapter/3365eed34374428492519f2b669aa33f/module/i4x://UI/Refresh/vertical/2908dbb7725146ffa4139b3e266ff9e9
`rijjaasif`

File: ooyala_player/public/js/ooyala_player.js (hunk 3 above)
Comment (moeez96) [reply ooyala_player/public/js/ooyala_player.js]: Confirm

File: ooyala_player/public/js/ooyala_player.js (hunk 1 above)
Comment (moeez96) [reply ooyala_player/public/js/ooyala_player.js]: removed
